docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,272 @@
+This page will focus on the neuron interactions and explain some of the relevant definitions from this Candid file.
+To keep up to date with new NNS release proposals from DFINITY, you can subscribe to [this forum thread](https://forum.dfinity.org/t/nns-updates-aggregation-thread/23551). 
+
+## Neuron attributes
+Before explaining how to modify a neuron, it is important to understand the most important parts of a neuron.
+Every user can stake ICP utility tokens into a neuron and pariticipate in the NNS DAO with this neuron.  
+
+* **Neuron ID**: Each neuron has an identity selected by NNS governance at neuron creation.
+* **Account**: Each neuron has an associated account on the ICP ledger where the locked ICP balance resides. This account is owned by the NNS governance canister and therefore a user cannot move staked tokens.
+* **Controller**: The principal that controls the neuron. The principal must identify a public key pair, which acts as a “master key,” such that the corresponding secret key should be kept very secure. A principal can control many neurons.
+
+* **Dissolve delay & dissolve state**: The tokens in a neuron are locked for a specified time, called the _dissolve delay_. This can be thought of like a kitchen timer that can only be turned in one direction. It can be arbitrarily increased, but only reduced by turning on the countdown and waiting for the time to pass. A neuron is eligible to vote if it has a dissolve delay of at least 6 months.
+  * A neuron can be _non-dissolving_ which means that the timer is stopped and the neuron's dissolve delay remains the same. 
+  * A neuron can be _dissolving_ which means that the timer is decreasing the neuron's dissolve delay as time passes. 
+  * Once the timer has counted down, a neuron is _dissolved_ and the ICP tokens can be unstaked again.
+* **Age**: Every non-dissolving neuron has an age, which denotes how long it has been in the non-dissolving state. 
+A neuron's dissolve delay and age increases the neuron's voting power. 
+
+* **Maturity**:  When neurons vote, over time the NNS increases their maturity to reward them. Maturity can be converted into ICP by spawning (see below).
+
+## Interacting with neurons
+All interactions with a neuron go through the following API on the NNS governance canister. This includes everything in a neuron's lifetime such as the neuron's creation, modifications and voting, and unstaking the ICP tokens.
+```
+type ManageNeuron = record {
+  id : opt NeuronId;
+  command : opt Command;
+  neuron_id_or_subaccount : opt NeuronIdOrSubaccount;
+};
+```
+
+### Creating a neuron and topping it up
+Staking ICP utility tokens in a neuron involves three steps:
+1. Compute the neuron's address on the ledger, where the staked tokens will be held. This corresponds to a subaccount of the NNS governance canister.
```

Comment:
NCR

we seriously should make this step easier I think. It would be great to have a website or a governance API that did this computation for you given a principal ID and a nonce